golangci.yml

```diff
@@ -106,6 +106,9 @@ linters-settings:
     - metres
     - kilometre
     - kilometres
+  revive:
```

Comment:
@dashpole we have revive [enabled](https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/pull/439/files#diff-02f3aab86a2dfffc9cf61ee7bd633da2c63efcd7a7c5d668932b93d7612d9dfaR118) but by default it looks like it doesn't do anything (https://github.com/mgechev/revive#configuration):

> By default revive will enable only the linting rules that are named in the configuration file

So we should look at the list of available checks to see if there are any others we should enable